batch/src/main/scala/uk/gov/ons/addressindex/readers/AddressIndexFileReader.scala

```diff
@@ -12,22 +12,124 @@ object AddressIndexFileReader {
 
   private lazy val config = ConfigFactory.load()
   private lazy val pathToCsv = config.getString("addressindex.files.csv.delivery-point")
+  private lazy val pathToBlpuCSV = config.getString("addressindex.files.csv.blpu")
+  private lazy val pathToClassificationCSV = config.getString("addressindex.files.csv.classification")
+  private lazy val pathToCrossrefCSV = config.getString("addressindex.files.csv.crossref")
+  private lazy val pathToLpiCSV = config.getString("addressindex.files.csv.lpi")
+  private lazy val pathToOrganisationCSV = config.getString("addressindex.files.csv.organisation")
+  private lazy val pathToStreetCSV = config.getString("addressindex.files.csv.street")
+  private lazy val pathToStreetDescriptorCSV = config.getString("addressindex.files.csv.street-descriptor")
+  private lazy val pathToSuccessorCSV = config.getString("addressindex.files.csv.successor")
 
   /**
     * Reads csv into a `DataFrame`
     *
     * @return `DataFrame` containing the delivery point data from CSV
     */
   def readDeliveryPointCSV(): DataFrame = SparkProvider.sqlContext.read
-      .format("com.databricks.spark.csv")
-      .schema(CSVSchemas.postcodeAddressFileSchema)
-      .option("header", "true") // Use first line of all files as header
-      .load(resolveAbsolutePath(pathToCsv))
+    .format("com.databricks.spark.csv")
```

Comment:
Create a private function that accepts Path and Schema and returns a CSV file (to avoid repetitions)